### DISCUSSION TITLE
Idea: All components designed to use children

### DISCUSSION THREAD

**Discussion by jpandersen87** [MetroStar/comet#75] [Ideas]
Either in addition to, or replacement of existing render logic, components should accept and render children. For example, the accordion component currently only accepts a javascript array of items to render instead of allowing the items to be passed manually as children (ex: `<Accordion><AccordionItem>...</AccordionItem></Accordion>`).

**Comment by jbouder** [comment]
I'm thinking allowing both a list as well as children would be the best approach. Created an issue here: https://github.com/MetroStar/comet/issues/78. Also, any chance you might be interested in taking this one on?

**Comment by jbouder** [comment]
@jpandersen87 , started work on this request here: https://github.com/MetroStar/comet/pull/79. Just wanted to verify this is what you're looking for. If so, i'll implement similarly on other applicable components.

**Reply by jbouder** [reply]
PR merged to support this request: https://github.com/MetroStar/comet/pull/79
